### DISCUSSION TITLE
How to connect symbols (pdb) from Cloud Smith

### DISCUSSION THREAD

**Discussion by maglini** [OrchardCMS/OrchardCore#9870] [Q&A]
![image](https://user-images.githubusercontent.com/60664635/124744892-e1cd9f80-df27-11eb-904b-00db97c5c456.png)

**Comment by ns8482e** [comment]
[Cloud smith limitations ](https://help.cloudsmith.io/docs/nuget-feed#current-limitations)
> Symbol packages are supported (and parsed), but symbol server is not yet supported.

**Reply by Skrypt** [reply]
@lskillen Any status update on this feature?

**Reply by lskillen** [reply]
Hey @Skrypt - we're currently working on NuGet-related functionality (upstreams and additional support). Let me see if I can have our team fold the symbols endpoints into this work, but I can't guarantee a date for it yet. Finger in the air estimation would be "sometime in August". :-)
